volumes/miovision/api/intersection_tmc.py

```diff
@@ -327,23 +340,20 @@ def pull_data(conn, start_time, end_time, intersection, path, pull, key, dupes):
             else:
                 logger.error('Could not successfully pull data for this intersection')
 
-        logger.info('Completed data pulling for %s', start_time)
-        try: 
-            insert_data(conn, start_time, end_iteration_time, table, dupes)
+        logger.info('Completed data pulling from {0:s} to {1:s}'
+                    .format(c_start_t, c_end_t))
+        try:
+            insert_data(conn, c_start_t, c_end_t, table, dupes)
         except psycopg2.Error as exc:
             logger.exception(exc)
             sys.exit(1)
-        
+
         if pull_data:
             logger.info('Skipping aggregating and processing volume data')
         else:
-            process_data(conn, start_time, end_iteration_time)
+            process_data(conn, c_start_t, c_end_t)
```

Comment:
Per our discussion, `process_data()` should get moved out of the `time_delta` loop and take `start_time, end_time` as parameters

Reply:
Moved code block out of the loop. Haven't made the requisite changes to `miovision_api.refresh_gapsize_lookup`, but as discussed we can save that for another PR.